azurerm/internal/services/cosmos/common/indexing_policy.go

```diff
@@ -85,6 +109,43 @@ func flattenCosmosDBIndexingPolicyExcludedPaths(input *[]documentdb.ExcludedPath
 	return excludedPaths
 }
 
+func flattenCosmosDBIndexingPolicyCompositeIndex(input []documentdb.CompositePath) []interface{} {
+	if input == nil {
+		return nil
+	}
+
+	indexPairs := make([]interface{}, 0)
+	for _, v := range input {
+		path := ""
+		if v.Path != nil {
+			path = *v.Path
+		}
+
+		block := make(map[string]interface{})
+		block["path"] = path
+		block["order"] = v.Order
```

Comment:
this is a Go Constant - so can we cast this:

```suggestion
		block["order"] = string(v.Order)
```